lisp/Makefile

```diff
@@ -76,7 +80,8 @@ julia-mode.el:
 	test -f ../etc/.IS.RELEASE || $(DOWNLOAD) \
 	https://raw.githubusercontent.com/JuliaEditorSupport/julia-emacs/master/julia-mode.el > $@
 
-ess-julia.elc: julia-mode.el julia-mode.elc
+ess.el:
+	sed -e 's/@@VERSION@@/$(ESSVERSION)/g' ess.el.in > $@
```

Comment:
With ESS and other elisp packages it's enough to just add the package to load-path and require. I want to develop on the source directly, not on some system folder. 

Plus, for many many years, till a few months ago ,we did recommend the add-to-load-path+(load ess-site). So I am pretty sure that loads of users who do not compile their ESS at all, let alone make install. They might not even have make on their machines. So, sorry, this template scheme is a bit too witty for us.

Reply:
Noted.  If "just adding the package to load-path and require" is officially supported, we should add a test in travis.  We also don't have a travis test for "downloading tarball and make install" either (this is harder to do since that installs to system directories).

Reply:
>  .. We also don't have a travis test for "downloading tarball and make install" either (this is harder to do since that installs to system directories).

No, it's easy if you know `make` very slightly better: e.g.,

    make install PREFIX=/tmp

should typically work an unixy system, hence travis.  Slightly more sophisticated would be

    TD=/tmp/rootESS$$; mkdir -p $TD; make install PREFIX=$TD

Reply:
A proper test ensures ESS works when emacs is started in a post `make install` environment. 
 It is easy enough to ensure an error-free tarballing into an arbitrary directory such as `/tmp`, (this is what #752 does), but stronger proof is required to claim ESS works when `make install` is issued with the systemwide defaults.